### ISSUE TITLE
Uncaught TypeError: Cannot read property 'paths' of undefined

### ISSUE THREAD

**Issue by gee-forr** [mehcode/atom-project-plus#42]
1. `$ apm install project-plus`
2. open atom
3. hit cmd+ctrl+p

notes: I've also attempted to disable auto-discovery and manually save projects, and I still get this problem. Hope you can help.

Hit me up if you need extra info to help debug.

**Atom Version**: 1.6.1
**System**: Mac OS X 10.11.4
**Thrown From**: [project-plus](https://github.com/mehcode/atom-project-plus) package, v0.8.0
### Stack Trace

Uncaught TypeError: Cannot read property 'paths' of undefined

```
At /Users/gabrielf/.atom/packages/project-plus/lib/provider/session.js:114

TypeError: Cannot read property 'paths' of undefined
    at /Users/gabrielf/.atom/packages/project-plus/lib/provider/session.js:114:34
    at FSReqWrap.readFileAfterClose [as oncomplete] (fs.js:380:3)
```
### Commands

```
     -1:09.8.0 project-plus:toggle-project-finder (atom-text-editor.editor.is-focused.vim-mode.normal-mode)
     -1:00.8.0 settings-view:open (atom-text-editor.editor.vim-mode.normal-mode.is-focused)
  2x -0:41.8.0 core:backspace (atom-text-editor#project-plus.projectHome.editor.mini.is-focused)
     -0:25.1.0 core:close (input#project-plus.showPath.)
     -0:23.9.0 project-plus:toggle-project-finder (atom-text-editor.editor.vim-mode.normal-mode.is-focused)
     -0:20.1.0 editor:consolidate-selections (atom-text-editor.editor.mini.is-focused)
     -0:20.1.0 core:cancel (atom-text-editor.editor.mini.is-focused)
     -0:08.9.0 command-palette:toggle (atom-text-editor.editor.vim-mode.normal-mode.is-focused)
     -0:03.5.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:03.5.0 project-plus:save (atom-text-editor.editor.vim-mode.normal-mode)
     -0:00.7.0 project-plus:toggle-project-finder (atom-text-editor.editor.vim-mode.normal-mode.is-focused)
```
### Config

``` json
{
  "core": {
    "disabledPackages": [
      "markdown-toc"
    ],
    "ignoredNames": [
      ".git",
      ".svn",
      ".DS_Store",
      "tmp",
      "bower_components",
      "log",
      "node_modules"
    ],
    "themes": [
      "one-dark-ui",
      "gruvbox"
    ]
  },
  "project-plus": {
    "autoDiscover": false,
    "projectHome": "/Users/gabrielf/projects",
    "showPath": true
  }
}
```
### Installed Packages

``` coffee
# User
atom-pair, v2.0.10
file-icons, v1.6.19
gruvbox, v1.0.5
linter, v1.11.4
linter-rails-best-practices, v0.2.2
linter-reek, v2.1.5
linter-rubocop, v0.4.7
project-plus, v0.8.0
relative-numbers, v0.5.1
vim-mode, v0.65.0

# Dev
No dev packages
```

**Comment by gee-forr** [comment]
Hey, just a quick update from my side.

I uninstalled and reinstalled the package, thinking it was something silly like that, but no dice. Still not working :( I've also since updated to Atom 1.6.2, and I'm getting the same error.

Hope it's something simple to fix. Please don't forget, hit me up anytime if you need extra debug info.

Thanks :)

**Comment by mehcode** [comment]
@gee-forr I haven't forgotten. I've just been otherwise occupied (my wife just gave birth to our first child 3 days ago). Heh. So I've been a bit busy.

One of these upcoming evenings I'll be working on OSS stuff (to be honest I do this mostly to decompress). I'll take a look then, unless @shemerey can look into it.

**Comment by gee-forr** [comment]
Oh wow! Congratulations man!!! My apologies, I didn't want to seem like I was pressuring you to fix this. I just wanted to give you a heads up.

Parenthood > Helping some internet rando with an editor plugin. Every single time.

Take care dude and look after your family, and thanks for all your hard work in the atom community. It's very appreciated.

**Comment by jccguimaraes** [comment]
@mehcode that's great! Enjoy everytime! it's worth it! :baby: 
I have a 15 month boy so I know the challenges to come to _our_ world!

**Comment by shemerey** [comment]
@gee-forr @mehcode travel a lot at this time, will try to take a look on a weekend, @mehcode congrats with a child

**Comment by mehcode** [comment]
@gee-forr Published v0.8.1 with a fix (possibly) -- re-open this if it still occurs for you.

**Comment by gee-forr** [comment]
Thanks! No longer receiving the error. Really appreciate it.

Hope all is going well with your new born son :)
